assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+          }
+        },
+        "reviews",
+        {
+          "render": {
+            "en": "toilets"
+          },
+          "question": {
+            "en": "Does this place have toilets?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "toilets=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has toilets"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "toilets=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have toilets"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "toilets"
+          }
+        },
+        {
+          "render": {
+            "en": "More details about this place: {description}"
+          },
+          "question": {
+            "en": "Would you like to add a general description of this place? (please keep it objective; opinions go into the reviews)"
+          },
+          "freeform": {
+            "key": "description",
+            "type": "text"
+          }
+        }
+      ],
+      "hideUnderlayingFeaturesMinPercentage": 0,
+      "icon": {
+        "render": "./assets/themes/campersites/caravan.svg",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "./assets/themes/campersites/caravan_green.svg"
+            }
+          }
+        ]
+      },
+      "width": {
+        "render": "8"
+      },
+      "iconSize": {
+        "render": "40,40,center",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=yes"
+              ]
+            },
+            "then": {
+              "en": "This place charges a fee"
+            }
+          },
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "You can use this place for free"
+            }
+          }
+        ]
+      },
+      "color": {
+        "render": "#00f",
+        "condition": {
+          "and": []
+        },
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "for free"
+            }
+          }
+        ]
+      },
+      "presets": [
+        {
+          "tags": [
+            "tourism=caravan_site"
+          ],
+          "title": {
+            "en": "camper site"
+          },
+          "description": {
+            "en": "Add a new official camper site. Please zoom in sufficiently for exact placement. Don't copy info from copyrighted content. Your data will be added straight to the global OpenStreetMap database!"
```

Comment:
These warnings are already included in the popup itself - they shouldn't be repeated. If you feel the current popup is not clear enough, open an issue or PR to fix that.

The description is meant to give a clarification on exactly what a camper site is, eventually with pictures.

FOr example, I added this on cyclofix as in some (non-english-speaking) countries, a "bicycle repair station" was confused with a "bicycle repair shop". It looks like this: 
![image](https://user-images.githubusercontent.com/1466478/110629403-68ee4f00-81a4-11eb-90ae-aee914155690.png)

Reply:
removed the unneeded parts and extended the description with more about the actual POI type